queue.h

```diff
@@ -14,14 +15,16 @@ enum dequeue_result {
   DEQUEUE_RESULT_EMPTY,
 };
 
-#define ARRAY_LENGTH(A) (sizeof(A)/sizeof((A)[0])) 
+#define ARRAY_LENGTH(A) (sizeof(A)/sizeof((A)[0]))
 
-#define QUEUE_DECLARATION(NAME, ITEM_TYPE, NUM_ITEMS)                                   \
```

Comment:
If you ever want to use this queue and have a queue handle within your own struct, it's helpful to have the queue type declaration separate from the declaration of the queue functions.